### ISSUE TITLE
[java] Possible False Positive On Catch Branch

### ISSUE THREAD

**Issue by davidgoate** [pmd/pmd#1270]
**Affects PMD Version:** 6.4.0

**Rule:** Identical catch branch

**Description:**
'catch' branch identical to 'CardException' branch

**Code Sample demonstrating the issue:**

```java
public Charge makePayment(final Chargeable chargeable, final UpFrontPaymentProvider provider) {
        try {
            final Map<String, Object> chargeParams = buildChargeParams(provider, chargeable);
            return Charge.create(chargeParams, options);
        } catch (final CardException e) {
            throw new PaymentCardException(e);
        } catch (final StripeException e) {
            throw new UnableToProcessPaymentException(e);
        }
    }
```

**Running PMD through:**  CLI

**Comment by oowekyala** [comment]
Duplicate of #1158, this was fixed in PMD 6.5.0
